contrib/endpoints/src/api_manager/config.cc

```diff
@@ -113,6 +113,36 @@ MethodInfoImpl *Config::GetOrCreateMethodInfoImpl(const string &name,
   return i->second.get();
 }
 
+bool Config::LoadQuotaRule(ApiManagerEnvInterface *env) {
```

Comment:
we can add this check in a separate CL

for a selector,  if there is a group_rule , don't use metric_rule